lib/resty/etcd/v3.lua

```diff
@@ -19,6 +20,7 @@ local decode_json   = cjson.decode
 local encode_json   = cjson.encode
 local encode_base64 = ngx.encode_base64
 local decode_base64 = ngx.decode_base64
+local semaphore = require("ngx.semaphore")
```

Comment:
It is better to align with the code format above.

Reply:
Changed.